### DISCUSSION TITLE
Gifts just disappear

### DISCUSSION THREAD

**Discussion by Janenutella** [Monika-After-Story/MonikaModDev#8422] [QA - tech support + general] [answered]
I've recently found out that you can give Monika gifts, and I tried giving her some but it won't work, they just disappear. There are some gifts that I'm certain works, like Hot Chocolate, Roses, and the Black Dress (But, with the black dress, she can't read the file)

I've tried giving her more stuff like the Thermos, the Pink Shirt, the Blue Shirt, and the Emerald Ribbon. But, every time I finish talking to her, I look back at the characters folder and the gift is gone.

Is it because it's a holiday? I don't know if I'm the only one with this problem.

An unedited video presentation
https://drive.google.com/file/d/14YUdTQIuUA8i6wAguBpnVRVTkMuhXegg/view?usp=sharing

**Comment by ThePotatoGuy** [comment, marked as answer]
Since it's currently christmas season, all gifts except consumables and certain special gifts are put under the tree and will be opened/reacted to on christmas. #6783 will add some sort of feedback so its clear that this happens.

**Reply by Janenutella** [reply]
Oh! Thank you so much for this answer! I really appreciate it! Once again, Thank you. I wish you a Merry Christmas!
